src/asmcnc/apps/drywall_cutter_app/widget_xy_move_drywall.py

```diff
@@ -1,11 +1,17 @@
+import sys, textwrap
```

Comment:
This looks like it could've been done creating a subclass of the existing widget_xy_move.py, understand that it's probably a time thing though. Happy to merge, but could you add a ticket to Jira in the backlog to look into? Just so it's not forgotten.